src/VisualStudio/CSharp/Impl/ChangeSignature/CSharpChangeSignatureViewModelFactoryService.cs

```diff
@@ -44,7 +44,9 @@ public override SymbolDisplayPart[] GeneratePreviewDisplayParts(AddedParameterVi
             return parts.ToArray();
         }
 
-        public override bool IsTypeNameValid(string typeName) => !SyntaxFactory.ParseTypeName(typeName).ContainsDiagnostics;
+#nullable enable
+        public override bool IsTypeNameValid(string typeName, ParseOptions? parseOptions) => !SyntaxFactory.ParseTypeName(typeName, options: parseOptions).ContainsDiagnostics;
+#nullable restore
```

Comment:
file is tiny.  i would prefer to nrt all of it or none of it.

Reply:
I'm going to do none. It implements an interface and I don't want to go annotating multiple levels here.